apps/src/code-studio/pd/application/teacher1819/Section4SummerWorkshop.jsx

```diff
@@ -182,40 +187,57 @@ export default class Section4SummerWorkshop extends ApplicationFormComponent {
             More details will be provided if you are accepted into the program.
           </h5>
         );
-      } else if (this.state.partnerWorkshops.length === 1) {
-        return (
-          <div>
-            <h5>
-              Your region’s assigned summer workshop will be
-              {` ${this.state.partnerWorkshops[0].dates} in`}
-              {` ${this.state.partnerWorkshops[0].location}.`}
-            </h5>
-
-            {this.renderAbleToAttendSingle()}
-          </div>
-        );
-      } else { // multiple workshops
-        const options = this.state.partnerWorkshops.map(workshop =>
-          `${workshop.dates} in ${workshop.location}`
-        );
-        options.push(NO_EXPLAIN);
-        const textFieldMap = {[NO_EXPLAIN]: 'explain'};
-        return this.dynamicCheckBoxesWithAdditionalTextFieldsFor(
-          "ableToAttendMultiple",
-          options,
-          textFieldMap
-        );
+      } else {
+        return this.renderPartnerWorkshops();
       }
     }
   }
 
+  renderPartnerWorkshops() {
+    let contents;
+    if (this.state.partnerWorkshops.length === 1) {
+      contents = (
+        <div>
+          <h5>
+            Your region’s assigned summer workshop will be
+            {` ${this.state.partnerWorkshops[0].dates} in`}
+            {` ${this.state.partnerWorkshops[0].location} `}
+            hosted by {` ${this.state.regionalPartnerName}.`}
+          </h5>
+
+          {this.renderAbleToAttendSingle()}
+        </div>
+      );
+    } else { // multiple workshops
+      const options = this.state.partnerWorkshops.map(workshop =>
+        `${workshop.dates} in ${workshop.location} hosted by ${this.state.regionalPartnerName}`
```

Comment:
is this what you didn't want, where we might have the same regional partner name repeated on every line?

Reply:
yes, but PM overrode my concern